schemas/adaptive-card.json

```diff
@@ -176,14 +281,57 @@
 					"version": "1.1",
 					"oneOf": [
 						{ "$ref": "#/definitions/Action.Submit" },
-						{ "$ref": "#/definitions/Action.OpenUrl" }
+						{ "$ref": "#/definitions/Action.OpenUrl" },
+						{ "$ref": "#/definitions/Action.ToggleVisibility" }
 					]
 				}
 			},
 			"required": [
 				"type"
 			]
 		},
+		"BackgroundImage": {
+			"type": "object",
+			"description": "Background image for the card or container.",
+			"properties": {
+				"url": {
+					"type": "string",
+					"description": "URL specifying the image to use as the background. Supports data URI."
+				},
+				"fillMode": {
+					"type": "string",
+					"description": "Controls how the background image fills the container or card.",
+					"enum": [
+						"cover",
+						"repeatHorizontally",
+						"repeatVertically",
+						"repeat"
+					]
+				},
+				"HorizontalAlignment": {
+					"type": "string",
+					"description": "Defines how the image should be aligned horizontally.",
+					"enum": [
+						"left",
+						"center",
+						"right"
+					]
+				},
+				"VerticalAlignment": {
+					"type": "string",
+					"description": "Defines how the image should be aligned vertically.",
+					"enum": [
+						"top",
+						"center",
+						"bottom"
+					]
+				}
+			},
+			"required": [
+				"url"
+			],
+			"version": "1.2"
+		},
 		"CardElement": {
```

Comment:
>"CardElement" [](start = 2, length = 13)

CardElement is missing the isVisible boolean property (related to ToggleVisibility)